docs/launching-remote-tests.md

```diff
@@ -0,0 +1,99 @@
+# Launching remote tests
+
+These tests can run remotely inside of Docker containers using Azure Container Instances.
+
+To understand this, it may help to read about how the runtime environment is organized [here](./runtime.md)
+
+In order to launch the app remotely, you need to:
+1. Build docker images for the device app and the service app
+2. Launch at least one service app with some unique `servicePool` name
+3. Launch at least one device app requesting to pair with that `servicePool`
+
+## Building docker images
+
+First, make sure your environment is set up.  Instructions are [here](./setting-up-your-thief-environment.md)
+
+Next, build the service image.  It requires a `source` and a `library_version`.
+The `source` defines "where to get the code"
+and the `library_version` defines "which version of the code to get from that source".
+
+The `source` is selected from a set of valid values, based on the langauge you are building.
+The format of the `library_version` is based on the selected `source`.
+
+We currently support the following sources:
+
+| langauge | source | language default? | library_version format |
+| - | - | - | - |
+| Python | pypi | yes |  package version.  e.g. "2.4.0" |
+
+The libraries are built using `scripts/build_image.sh`.
+
+```
+USAGE: ./build-image.sh [--platform platform] --langauge language_short_name --library library [--source library_source] --version library_version [--tag extra_tag]
```

Comment:
What if I want to use .NET service client and NodeJS device client?

Reply:
that's possible.  build-image.sh builds either the device or the service app, so you would just adjust the parameters to build/run the mix.